### ISSUE TITLE
General questions about parameters in the pre-trained model

### ISSUE THREAD

**Issue by FlorianeSchreiber** [remicres/sr4rs#43]
Hello Rémi,

Hope you are doing well. I have several general questions, notably about the parameters you used for your pre-trained model. 
If I understood well, you used 15k patchs and the number of pixels are : LR : 128x128 ; HR : 512 x 512, and a batch size of 4 ? Is that right ? 
Approximately, how many epochs did you use ? (~10, ~50, ~100, ~200, more ?)
What are the weights you used for the VGGnet ? In the literature they often use "imagenet" weights, and I would know if you use this one, or another one. 

Thank you 😊 

Floriane

**Comment by remicres** [comment]
Hi, 

Here is what we used:
- LR: 64
- HR: 256
- Batch size: 4
- 150k patches
- Don't remember the number of epochs, or learning rate. Best is to experiment with your own data. The "good looking" thing is very subjective, so there isn't a universal recipe here :) stop the training when the network doesn't change the results.
